uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -1204,3 +1204,99 @@ Feature: GGMQ-1
+    And I create client device "publisher"
+    And I create client device "subscriber"
+    When I associate "subscriber" with ggc
+    When I associate "publisher" with ggc
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.clientdevices.Auth configuration to:
+    """
+{
+    "MERGE":{
+        "deviceGroups":{
+            "formatVersion":"2021-03-05",
+            "definitions":{
+                "MyPermissiveDeviceGroup":{
+                    "selectionRule":"thingName: ${publisher} OR thingName: ${subscriber}",
+                    "policyName":"MyPermissivePolicy"
+                }
+            },
+            "policies":{
+                "MyPermissivePolicy":{
+                    "AllowAll":{
+                        "statementDescription":"Allow client devices to perform all actions.",
+                        "operations":[
+                            "*"
+                        ],
+                        "resources":[
+                            "*"
+                        ]
+                    }
+                }
+            }
+        }
+    }
+}
+    """
+    And I update my Greengrass deployment configuration, setting the component <agent> configuration to:
+    """
+{
+    "MERGE":{
+        "controlAddresses":"${mqttControlAddresses}",
+        "controlPort":"${mqttControlPort}"
+    }
+}
+    """
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 5 minutes
+    And the aws.greengrass.clientdevices.mqtt.EMQX log on the device contains the line "is running now!." within 1 minutes
+
+    Then I discover core device broker as "localMqttBroker1" from "publisher" in OTF
+    Then I discover core device broker as "localMqttBroker2" from "subscriber" in OTF
+    And I connect device "publisher" on <agent> to "localMqttBroker1" using mqtt "<mqtt-v>"
+    And I connect device "subscriber" on <agent> to "localMqttBroker2" using mqtt "<mqtt-v>"
+
+    And I set MQTT user property with key "type" and value "json" to transmit
+    And I set MQTT user property with key "type" and value "json" to receive
+
+    When I subscribe "subscriber" to "iot_data_1" with qos 0
+    When I publish from "publisher" to "iot_data_1" with qos 0 and message "Hello world"
+    And message "Hello world" received on "subscriber" from "iot_data_1" topic within 5 seconds
+
+    And I clear message storage
+    And I clear MQTT user properties to transmit
+    And I clear MQTT user properties to receive
+    And I set MQTT user property with key "agent-control" and value "control" to transmit
+    When I subscribe "subscriber" to "iot_data_2" with qos 0
+    When I publish from "publisher" to "iot_data_2" with qos 0 and message "Hello world1"
+    And message "Hello world1" is not received on "subscriber" from "iot_data_2" topic within 5 seconds
```

Comment:
Why is not received ?
When `received` properties is null that means no filter for properties is applied.